modules/serverless-kafka-source-kn.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+[id="serverless-kafka-source-kn_{context}"]
+= Creating a Kafka event source using the `kn` CLI
+
+This section describes how to create a KafkaSource using the `kn` command.
+
+:FeatureName: Creating a Kafka event source using the `kn` CLI
+include::../modules/technology-preview.adoc[leveloffset=+0]
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, Knative Serving, and the `KnativeKafka` custom resource definitions are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have access to a Kafka cluster that produces the Kafka messages you want to import.
```

Comment:
Call out that it's specifically an AMQ Streams cluster maybe since that's the only Kafka we support?

Reply:
Done, but wrote "Kafka cluster" as an alternative name to be used for the rest of the doc (a-la an abbreviation). "Red Hat AMQ Streams cluster" is too clunky for multiple mentions IMO.